javascript-if/main.js

```diff
@@ -0,0 +1,79 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'j') {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
This should only return `true` if `string` begins with a _capital_ "J', not a _lowercase_ "j".

---

![image](https://user-images.githubusercontent.com/16143771/169149518-0e56f9f0-82c4-4ff0-a658-9d5be8009cbc.png)